### ISSUE TITLE
All downloads fail to commence from NHK World; unable to download JSON metadata

### ISSUE THREAD

**Issue by JasonWYLeung** [ytdl-org/youtube-dl#31183]
## Checklist

- [x] I'm reporting a broken site support
- [x] I've verified that I'm running youtube-dl version **2021.12.17**
- [x] I've checked that all provided URLs are alive and playable in a browser
- [x] I've checked that all URLs and arguments with special characters are properly quoted or escaped
- [x] I've searched the bugtracker for similar issues including closed ones

## Verbose log

```
[debug] System config: []
[debug] User config: []
[debug] Custom config: []
[debug] Command-line args: ['https://www3.nhk.or.jp/nhkworld/en/ondemand/video/4001316/', '-v']
[debug] Encodings: locale UTF-8, fs utf-8, out utf-8, pref UTF-8
[debug] youtube-dl version 2021.12.17
[debug] Python version 3.10.6 (CPython) - macOS-12.5-x86_64-i386-64bit
[debug] exe versions: ffmpeg 5.1, ffprobe 5.1, rtmpdump 2.4
[debug] Proxy map: {}
[NhkVod] 4001-316: Downloading JSON metadata
ERROR: Unable to download JSON metadata: HTTP Error 503: Service Temporarily Unavailable (caused by <HTTPError 503: 'Service Temporarily Unavailable'>); please report this issue on https://yt-dl.org/bug . Make sure you are using the latest version; see  https://yt-dl.org/update  on how to update. Be sure to call youtube-dl with the --verbose flag and include its complete output.
  File "/usr/local/Cellar/youtube-dl/2021.12.17/libexec/lib/python3.10/site-packages/youtube_dl/extractor/common.py", line 634, in _request_webpage
    return self._downloader.urlopen(url_or_request)
  File "/usr/local/Cellar/youtube-dl/2021.12.17/libexec/lib/python3.10/site-packages/youtube_dl/YoutubeDL.py", line 2288, in urlopen
    return self._opener.open(req, timeout=self._socket_timeout)
  File "/usr/local/Cellar/python@3.10/3.10.6_1/Frameworks/Python.framework/Versions/3.10/lib/python3.10/urllib/request.py", line 525, in open
    response = meth(req, response)
  File "/usr/local/Cellar/python@3.10/3.10.6_1/Frameworks/Python.framework/Versions/3.10/lib/python3.10/urllib/request.py", line 634, in http_response
    response = self.parent.error(
  File "/usr/local/Cellar/python@3.10/3.10.6_1/Frameworks/Python.framework/Versions/3.10/lib/python3.10/urllib/request.py", line 563, in error
    return self._call_chain(*args)
  File "/usr/local/Cellar/python@3.10/3.10.6_1/Frameworks/Python.framework/Versions/3.10/lib/python3.10/urllib/request.py", line 496, in _call_chain
    result = func(*args)
  File "/usr/local/Cellar/python@3.10/3.10.6_1/Frameworks/Python.framework/Versions/3.10/lib/python3.10/urllib/request.py", line 643, in http_error_default
    raise HTTPError(req.full_url, code, msg, hdrs, fp)

```

## Description

Videos from NHK's site fail to download; instead, an HTTP 503 error is given. 
The video can be downloaded using the workaround described in [this issue](https://github.com/ytdl-org/youtube-dl/issues/31028).

I have tried this with my Mac, and all downloads from NHK World VOD show an HTTP 503 Error. 

Some other failing URLs:
https://www3.nhk.or.jp/nhkworld/en/ondemand/video/2085075/
https://www3.nhk.or.jp/nhkworld/en/ondemand/video/2015285/
https://www3.nhk.or.jp/nhkworld/en/ondemand/video/2032240/

Downloading these URLs display an HTTP 503 error, with the verbose output similar to the one above.

**Comment by dirkf** [comment]
Please try the git master which has the patch from #31028:
```
$ python -m youtube_dl -v -F 'https://www3.nhk.or.jp/nhkworld/en/ondemand/video/4001316/'
[debug] System config: [u'--prefer-ffmpeg']
[debug] User config: []
[debug] Custom config: []
[debug] Command-line args: [u'-v', u'-F', u'https://www3.nhk.or.jp/nhkworld/en/ondemand/video/4001316/']
[debug] Encodings: locale UTF-8, fs UTF-8, out UTF-8, pref UTF-8
[debug] youtube-dl version 2021.12.17
[debug] Git HEAD: 380c7e935
[debug] Python version 2.7.17 (CPython) - Linux-4.4.0-210-generic-i686-with-Ubuntu-16.04-xenial
[debug] exe versions: avconv 4.3, avprobe 4.3, ffmpeg 4.3, ffprobe 4.3
[debug] Proxy map: {}
[NhkVod] 4001-316: Downloading JSON metadata
[Piksel] nw_vod_v_en_4001_316_20220814001000_01_1660406534: Downloading webpage
[Piksel] nw_vod_v_en_4001_316_20220814001000_01_1660406534: Downloading JSON metadata
[Piksel] nw_vod_v_en_4001_316_20220814001000_01_1660406534: Downloading JSON metadata
[Piksel] d7644373: Downloading m3u8 information
[Piksel] d7644373: Downloading SMIL file
[info] Available formats for d7644373:
format code  extension  resolution note
http-381     mp4        320x180     381k video@ 255k, audio@126k, 123.93MiB
hls-390      mp4        unknown     390k 
http-390     mp4        320x?       390k 
http-662     mp4        640x360     662k video@ 536k, audio@126k, 214.16MiB
hls-677      mp4        unknown     677k 
http-677     mp4        640x?       677k 
http-1241    mp4        1280x720   1241k video@1115k, audio@126k, 400.49MiB
hls-1270     mp4        unknown    1270k 
http-1270    mp4        1280x?     1270k 
http-2124    mp4        1280x720   2124k video@1998k, audio@126k, 684.68MiB
http-2174    mp4        1280x?     2174k 
http-10267   mp4        1920x1080  10267k video@10267k, audio@  0k, 3.30GiB (best)
$
```
When NHK first changed the API URL, we got bad data back; now we get 503.

**Comment by JasonWYLeung** [comment]
Thanks for the patch! It works when downloading the video
